lib/jsonapi/view.ex

```diff
@@ -140,6 +142,15 @@ defmodule JSONAPI.View do
         def namespace, do: Application.get_env(:jsonapi, :namespace, "")
       end
 
+      def pagination_links(data, conn, page) do
+        paginator = Application.get_env(:jsonapi, :paginator, @paginator)
+        if  Code.ensure_loaded?(paginator) && function_exported?(paginator, :paginate, 4) do
```

Comment:
Extra whitespace after the `if`

Reply:
Fixed in latest commit, also catched by `mix format`